regenerate.py

```diff
@@ -4,77 +4,64 @@
 
 import argparse
 import glob
-import itertools
 import logging
 import os
 import subprocess
 
 logger = logging.getLogger(__name__)
 LOGGING_LEVELS = {
-  'NOTSET': logging.NOTSET,
-  'DEBUG': logging.DEBUG,
-  'INFO': logging.INFO,
-  'WARNING': logging.WARNING,
-  'ERROR': logging.ERROR,
-  'CRITICAL': logging.CRITICAL,
+    'NOTSET': logging.NOTSET,
```

Comment:
What went wrong with the formatting here? Is it an issue with the autoformatter?

Reply:
Might be a formatter settings error (i.e. in `.style.yapf`)? I think `yapf` by default more deeply indents multi-line lists and dicts. Don't have a problem with it myself, but I'm happy to look into the settings if you like.

Reply:
I'd like to avoid a situation where we are both using yapf, and we have two different styles.

Is your yapf using the .style.yapf that's in the repo? That one has a max line length of 80 chars, and I think some of the lines in this PR exceed that limit.

Reply:
Whoops, I forgot that I have some settings overridden for my yapf config. I'll run making sure to use the .style.yapf in the repo.